en/docs/setup/configuring-a-jdbc-user-store.md

```diff
@@ -171,36 +171,32 @@ those properties as follows.
 Add the following configuration to `<IS-HOME>/repository/conf/deployment.toml`.
 
 ``` toml
-[user_store.properties]
+[user_store]
```

Comment:
since we have given the user store config example below in line 180, is this needed here?